web/gui/dashboard_info.js

```diff
@@ -4274,6 +4274,47 @@ netdataDashboard.context = {
         info: 'Rate at which segments are sent, including those on current connections, but excluding those containing only retransmitted bytes.'
     },
 
+    // ------------------------------------------------------------------------
+    // WMI (IIS)
+
+    'wmi.iis_website_traffic': {
+        info: 'The total bytes of data that have been sent and received by the web service since the service started.'
+    },
+    'wmi.iis_website_requests_rate': {
+        info: 'The number of requests that have been made since the web service was started.'
```

Comment:
We display as rate, because we should counts/s.

Reply:
But this doesn't make sense.. if the description says something is a count of something since the service started but we show it as a rate. 

Description and what we show in chart should match.

Reply:
If we add the words `per second` after counter, I think we keep the main message and we solve the issue you raised.

Reply:
I added the cited words with last commits, please, take a look.